assets/chezmoi.io/docs/reference/templates/1password-functions/onepasswordRead.md

```diff
@@ -19,5 +19,14 @@ interactively prompted to sign in.
     is equivalent to calling
 
     ```console
-    $ op read op://vault/item/field
+    $ op read --no-newline op://vault/item/field
+    ```
+
+!!! info
```

Comment:
You can remove this `!!! info` block. For the sanity of the maintainers, the chezmoi documentation is only valid for the current version of chezmoi. Otherwise, every change would have to be prefixed with "versions of chezmoi prior to X.Y.Z don't have this...".

Reply:
Got it. It *may* be worth considering adding a `since` block to various new features like Elixir’s documentation does: https://hexdocs.pm/elixir/Version.html#parse_requirement!/1, especially when there have been some really neat features added. We wouldn’t necessarily document the changes as you indicated here, but if we wanted to *remove* a template in the future, we could also mark those as `deprecated since…`.